digitalstrategie/assets/scss/components/_books.scss

```diff
@@ -0,0 +1,15 @@
+.chapter-link {
+  & a {
+    color: $text-gray;
+    text-decoration: none;
+  }
+}
+
+.chapter-link--active {
```

Comment:
Hmm, to be a variant, the form without the variant should be used with it. But that's a bit annoying here as they only share the `text-decoration: none;`. I would be alright with keeping it like that for lack of better naming ideas.